_includes/header.html

```diff
@@ -29,17 +29,45 @@
             <a class="nav-link" href="https://why.isthmuscarshare.com">Why Carshare?</a>
           </li>
           <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown2" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <button class="nav-link dropdown-toggle" id="navbarDropdown2" type="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
               Contact
-            </a>
+            </button>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown2">
               <!-- <a class="dropdown-item" href="#">Trip Feedback</a> -->
               <a class="dropdown-item" href="FAQ.html">FAQ</a>
               <a class="dropdown-item" href="join.html">Join Now</a>
             </div>
           </li>
+          <li class="nav-item" id="liLoginLink">
+              <a class="nav-link" href="login.html">Login</a>
+          </li>
+          <li class="nav-item" id="liDisplayUserName" style="display: none;">
+              <div>
+                  <p id="pDisplayUserName"></p>
+                  <a id="aLogoutLink" class="nav-link">Logout</a>
+              </div>
+          </li>
         </ul>
       </div>
     </nav>
   </div>
-</header>
\ No newline at end of file
+</header>
+<script type="module">
+    import { updateName, signOut } from "./assets/js/authentication.js"
+    const name = updateName(name => {
```

Comment:
This took me forever to wrap my head around.  The one thing I'm still not getting is whether it's important for that to be returning the `const name`.  I think that's a different "name" than the one defined inside `updateName` which is then passed to this anonymous function.

Reply:
I can fix this up. It's actually totally unnecessary to assign this to a const, I just didn't clean it up after I tried a few different things. I can see why you would be confused!